src/Domain/FaceNormal.hpp

```diff
@@ -43,13 +46,13 @@ class Mesh;
  */
 template <size_t VolumeDim, typename TargetFrame>
 tnsr::i<DataVector, VolumeDim, TargetFrame> unnormalized_face_normal(
-    const Mesh<VolumeDim - 1>& interface_mesh,
+    const domain::Mesh<VolumeDim - 1>& interface_mesh,
     const ElementMap<VolumeDim, TargetFrame>& map,
     const Direction<VolumeDim>& direction) noexcept;
```

Comment:
Why does `Mesh` get a namespace but not `ElementMap` or `Direction`?  Similar things happen in other places as well.

Reply:
Because it's much easier to namespace one class at a time rather than 30....